webapp/.env.example

```diff
@@ -72,3 +72,8 @@ REACT_APP_MKR_MAX_CONVERSION_AMOUNT=
 REACT_APP_SNT_MAX_CONVERSION_AMOUNT=
 REACT_APP_ZIL_MAX_CONVERSION_AMOUNT=
 REACT_APP_RCN_MAX_CONVERSION_AMOUNT=
+
+# Redux
+
+REACT_APP_REDUX_ACTION_ONLY_TYPE=
```

Comment:
I think this should be called something like:

`REACT_APP_REDUX_ACTION_ONLY_DISPLAY_TYPE`
`REACT_APP_REDUX_ACTION_ONLY_SHOW_TYPE`
`REACT_APP_REDUX_ACTION_SKIP_PAYLOAD`

I thought it was a flag for letting only some types through